actors/builtin/reward/reward_state.go

```diff
@@ -23,9 +23,9 @@ type State struct {
 
 type AddrKey = adt.AddrKey
 
-// These numbers are placeholders, but should be in units of attoFIL
-var SimpleTotal, _ = big.FromString("1000000000000000000000000")
-var BaselineTotal, _ = big.FromString("1000000000000000000000000")
+// These numbers are placeholders, but should be in units of attoFIL, 10^-18 FIL
+var SimpleTotal, _ = big.FromString("100000000000000000000000000") // 100M for testnet, PARAM_FINISH
```

Comment:
Use `big.Mul(big.NewInt(100e6), big.NewInt(1e18))` and then you wont need any comments about what they mean.